app/Models/User.php

```diff
@@ -68,6 +70,16 @@ class User extends SnipeModel implements AuthenticatableContract, AuthorizableCo
         'company_id'   => 'integer',
     ];
 
+
+    protected $dates = [
+        'created_at',
```

Comment:
This isn't a 'date' it's a timestamp, so I'm not sure that you want this here. Also this is mostly handled by the Timestamps attribute within various models, so I don't know that redeclaring it here is a good idea.

Reply:
Casting things as dates simply makes them grokable via Carbon without us having to instantiate them separately.